website/source/docs/providers/aws/r/autoscaling_notification.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_vpc"
+sidebar_current: "docs-aws-resource-autoscaling-notification"
+description: |-
+  Provides an AutoScaling Group with Notification support
+---
+
+# aws\_vpc
+
+Provides an AutoScaling Group with Notification support, via SNS Topics
```

Comment:
If I understand correctly this feature actually abstracts creating multiple AWS resources at the same time from the same configuration "template". We should note that here to make it clear that this does not map 1:1 to what you'd see the API.